### PR TITLE
Added Writers badge to handbook

### LINKED ISSUE #3533: [Handbook]Add the writers program UX badge to the  file
**Current Behavior**
Following up on issue https://github.com/layer5io/layer5/issues/3527 we would like to add the writers ux badge to the  "[Recognitions](https://layer5.io/community/handbook/community)" in handbook.

**Desired Situation**

Add the [writers ux badge](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=1690%3A10102&t=BDGY3h6mSVmZiBAn-1) to the Recognitions section for the handbook.

---
[Optional] **Alternatives**

208754875-e20e0bb7-7ec7-4b22-b136-4319e7a888a9

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Savio629**
Signed-off-by: Savio629 <diassavio629@gmail.com>

**Description**

This PR fixes #3533

**Notes for Reviewers**
@Anita-ihuman 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 987d003e51583abd3e9e3e36ae0f47c0c0e4bf21 at: https://63d970324d6e6921401b8401--layer5.netlify.app

**Review by Anita-ihuman — COMMENTED** [review]
Thank you @Savio629 for adding this badge here.
It looks great so far. We should address the failed checks

**Review by UtkarshMishra12 — APPROVED** [review]
LGTM Thanks @Savio629 🚀🥇

**Review by eeshaanSA — APPROVED** [review]
Lgtm

File: src/sections/Community/Handbook/community.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import NightHawkLogo from "../../../assets/images/nighthawk/icon-only/SVG/nighth
 import TocPagination from "../../../components/handbook-navigation/TocPagination";
 import IntraPage from "../../../components/handbook-navigation/intra-page";
 import uiuxrLogo from "../../../assets/images/uiuxr/uiuxr.svg";
+import writersLogo from "../../../assets/images/writer-program/writer-program-badge.svg";
```
Comment (leecalcote) [inline src/sections/Community/Handbook/community.js:18]: @eeshaanSA will you please see that this is moved to an appropriate folder containing other community badges?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/sections/Community/Handbook/community.js (hunk 1 above)
Comment (Savio629) [reply src/sections/Community/Handbook/community.js:18]: Actually @leecalcote , the svg file was initially present in the folder. I just had to link it in handbook.

**Comment by leecalcote** [comment]
@Savio629 thanks for this PR. We _just_ split this section into two pages. Please see #3724. Given this change, let's close this PR and encourage you to review the new Recognitions page, making any changes that you deem necessary based on the new content.

**Pull request closed without merging**